media-libs/opus/opus-1.3.1-r2.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic multilib-minimal
+
+DESCRIPTION="Open codec for interactive speech and music transmission over the Internet"
+HOMEPAGE="https://opus-codec.org/"
+SRC_URI="https://archive.mozilla.org/pub/opus/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sparc ~x86"
+INTRINSIC_FLAGS="cpu_flags_x86_sse cpu_flags_arm_neon"
+IUSE="custom-modes doc static-libs ${INTRINSIC_FLAGS}"
+
+BDEPEND="doc? (
+		app-doc/doxygen
+		media-gfx/graphviz
+	)"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-1.3.1-libdir-macro.patch
+)
+
+multilib_src_configure() {
+	local myeconfargs=(
+		$(use_enable custom-modes)
+		$(use_enable doc)
+		$(use_enable static-libs static)
```

Comment:
sure?

Reply:
static-libs or whether or not I'd actually fixed the bug? I've fixed the latter now, but we can see if removing it is possible.